### PR TITLE
Badges

### PR DESCRIPTION
**Pull request by JatuFaique**
Please review my badge component,
PS: Avatars have been merged alr
Screenshot 2022-02-03 at 12-36-19 Badges
eady from avatar branch.

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  flex-wrap: wrap;
+  padding: 1rem 0;
+  gap: 2px;
+}
+
+.avatar-section {
+  border: 1px solid black;
+}
+
+.bs {
```
Comment (zabihshaik) [inline components/avatar/avatar.css:33]: Better to use understandable class names

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  border: 1px solid var(--border-succes);
+}
+
+.av-m {
+  position: relative;
+  /* height: 10%;
+  width: 10%; */
+  height: 3em;
+  width: 3em;
+  font-size: 16px;
```
Comment (zabihshaik) [inline components/avatar/avatar.css:55]: Better to use rems instead of px

File: components/avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./avatar.css">
+
+    <title>Avatar</title>
+</head>
+<body>
+    <h2 class='sub-heading'>Avatars Medium</h2>
+    <section class="avatar-section disp-flex">
+        <div class="av-s bs">A</div>
```
Comment (zabihshaik) [inline components/avatar/avatar.html:14]: Better to use understandable class names

File: components/badge/badges.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../avatar/avatar.css">
```
Comment (zabihshaik) [inline components/badge/badges.html:7]: Better to add a meta tag with author as name and your name in content

**Review by zabihshaik — COMMENTED** [review]
Everything looks Good. But a few suggestions:
1. Better to use understandable class names 
2. Better to use rems instead of px
3. Using meta tag with your name

**Comment by naazneen** [comment]
Seems fine.

**Review by naazneen — COMMENTED** [review]
Good to go

**Pull request merged**